include/tester/db/VTDatabase.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "Database.hpp"
+
+class VTDatabase : Database
+{
+public:
+    VTDatabase(const std::string &domain):
+        Database(domain)
+    {     
+    }
+
+    ~VTDatabase() = default;
+
+    void save(const bsoncxx::document::value &document)
+    {
+        Database::save(COLLECTION_NAME ,document);
```

Comment:
instead of `COLLECTION_NAME`, how about `TEST_LOGS_COLLECTION`?

Reply:
Also, read my comment below on the `Database::` prefix - I think its redundant if you'll make `Database::save` use the `protected` modifier.

Reply:
Done